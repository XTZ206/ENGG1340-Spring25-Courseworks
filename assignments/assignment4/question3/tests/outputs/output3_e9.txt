[2] 0
[4] 0 9
[4] 9
