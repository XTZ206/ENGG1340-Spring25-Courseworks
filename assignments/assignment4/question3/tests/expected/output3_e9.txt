[2] 0
[4] 0 6
[4] 0 6 8
[4] 0 -2
[4] 2
[4] 2 7
[4] -5
[4] -5 6
[4] 1
[4] 1 5
[4] 1 5 4
[4] 1 1
[4] 1 1 6
[4] 1 -5
[4] -4
[4] -4 4
[4] 0
